### ISSUE TITLE
how to set the minimum learning rate for cosine lr_scheduler?

### ISSUE THREAD

**Issue by xubuvd** [huggingface/open-muse#144]
@dataclass
class TrainingArguments(transformers.TrainingArguments):
    gradient_checkpointing_kwargs={'use_reentrant':False}
    lr_scheduler_kwargs={
        "eta_min":1e-6,
        "num_cycles":1,
    }

It did not work. how to set the minimum learning rate in transformers-4.51.3?
